plugin/src/main/java/com/redhat/jenkins/plugins/ci/pipeline/CIMessageSenderStep.java

```diff
@@ -82,7 +92,8 @@ public CIMessageSenderStep(final String providerName,
                                final MESSAGE_TYPE messageType,
                                final String messageProperties,
                                final String messageContent,
-                               Boolean failOnError) {
+                               Boolean failOnError,
+                               Integer timeToLiveMinutes) {
```

Comment:
IMO this needs to be added in https://github.com/jenkinsci/jms-messaging-plugin/blob/master/plugin/src/main/resources/com/redhat/jenkins/plugins/ci/pipeline/CIMessageSenderStep/config.jelly as well, no?

Reply:
@olivergondza , that change is already in the PR, unless I am missing something.